src/main/java/org/apache/maven/shared/utils/logging/MessageUtils.java

```diff
@@ -98,7 +98,14 @@ private static void doSystemUninstall()
     {
         if ( JANSI )
         {
-            AnsiConsole.systemUninstall();
+            try
+            {
+                AnsiConsole.systemUninstall();
+            }
+            catch ( RuntimeException ex )
```

Comment:
does it throw a specific runtime exception?

Reply:
No.